src/pathfinder/aystar.cpp

```diff
@@ -84,24 +84,24 @@ void AyStar::CheckTile(AyStarNode *current, PathNode *parent)
  * This function is the core of %AyStar. It handles one item and checks
  * its neighbour items. If they are valid, they are added to be checked too.
  * @return Possible values:
- *  - #AYSTAR_EMPTY_OPENLIST : indicates all items are tested, and no path has been found.
- *  - #AYSTAR_LIMIT_REACHED : Indicates that the max_search_nodes limit has been reached.
- *  - #AYSTAR_FOUND_END_NODE : indicates we found the end. Path_found now is true, and in path is the path found.
- *  - #AYSTAR_STILL_BUSY : indicates we have done this tile, did not found the path yet, and have items left to try.
+ *  - #AyStarStatus::EmptyOpenList : indicates all items are tested, and no path has been found.
+ *  - #AyStarStatus::LimitReached : Indicates that the max_search_nodes limit has been reached.
+ *  - #AyStarStatus::FoundEndNode : indicates we found the end. Path_found now is true, and in path is the path found.
+ *  - #AyStarStatus::StillBusy : indicates we have done this tile, did not found the path yet, and have items left to try.
```

Comment:
Worth cleaning up the comments at the same time? These basically just duplicate the enum docs

Reply:
Just leave it as the possible return values?(Not that we ever list that for anything else as far as I'm aware...)